api/src/libs/migrationUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { BusinessesDataClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { UserData } from "@shared/userData";
+
+export async function pushBusinessesInUsersDataToBusinessTable(
+  userData: UserData,
+  businessesDataClient: BusinessesDataClient,
+  logger: LogWriterType
+): Promise<void> {
+  for (const businessId in userData.businesses) {
+    logger.LogInfo(`Starting migration for business ID ${businessId} for user ${userData.user.id}`);
+    const businessData = userData.businesses[businessId];
+    try {
+      await businessesDataClient.put({
```

Comment:
left a cemment in the file where the put is created, the migration fn would lead to an error